Domains/4-Application/ISM/IntegratedStructuralModel.ecschema.xml

```diff
@@ -5,7 +5,7 @@
     <ECSchemaReference name="BisCore" version="01.00.14" alias="bis" />
     <ECSchemaReference name="Analytical" version="01.00.00" alias="anlyt" />
     <ECSchemaReference name="AecUnits" version="01.00.03" alias="AECU" />
-    <ECSchemaReference name="StructuralAnalysis" version="01.00.02" alias="sa" />
+    <ECSchemaReference name="StructuralAnalysis" version="01.00.04" alias="sa" />
```

Comment:
Same comment about the version of a WIP copy being ahead of the latest released. Thus, it should be 1.0.5 in this case.